scripting/index.rst

```diff
@@ -10,4 +10,5 @@ Introduction to Scripting
    usage
    event-groups
    optimization
+   tracing-events
```

Comment:
nit: I'd personally prefer this go above optimization since it's tangentially related to event-groups